prow/gerrit/adapter/adapter_test.go

```diff
@@ -1111,8 +1208,25 @@ func TestProcessChange(t *testing.T) {
 				gc:            &gc,
```

Comment:
I think we should also test scenario where `gc = nil`

Reply:
Done